langchain-core/src/token_usage.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Shared interface for token usage
+ * return type from LLM calls.
+ */
+export interface TokenUsage {
```

Comment:
Should export from `base_language` instead?

Reply:
Ah yeah that's probably better. Wasn't 100% on where to put this. ty